packages/ui/src/components/Accordion/Accordion.stories.tsx

```diff
@@ -35,30 +36,93 @@ const Template: ComponentStory<typeof Accordion.Root> = ({
   return (
     <Accordion.Root {...rest}>
       <Accordion.Item value="one">
-        <Accordion.Trigger Icon={AcademicCapIcon} subtitle="Subtitle">
-          Accordion One
-        </Accordion.Trigger>
+        <Accordion.Header>
+          <Accordion.Trigger>Accordion One</Accordion.Trigger>
+        </Accordion.Header>
         <Accordion.Content>{children}</Accordion.Content>
       </Accordion.Item>
       <Accordion.Item value="two">
-        <Accordion.Trigger Icon={Cog8ToothIcon} subtitle="Subtitle">
-          Accordion Two
-        </Accordion.Trigger>
+        <Accordion.Header>
+          <Accordion.Trigger>Accordion Two</Accordion.Trigger>
+        </Accordion.Header>
         <Accordion.Content>{children}</Accordion.Content>
       </Accordion.Item>
       <Accordion.Item value="three">
-        <Accordion.Trigger Icon={GlobeAmericasIcon} subtitle="Subtitle">
-          Accordion Three
-        </Accordion.Trigger>
+        <Accordion.Header>
+          <Accordion.Trigger>Accordion Three</Accordion.Trigger>
+        </Accordion.Header>
         <Accordion.Content>{children}</Accordion.Content>
       </Accordion.Item>
     </Accordion.Root>
   );
 };
 
-const Text = () => {
-  faker.seed(0);
-  return <p>{faker.lorem.sentences(5)}</p>;
+const TemplateWithCustomHeader: ComponentStory<typeof Accordion.Root> = ({
+  children,
+  ...rest
+}) => {
+  return (
+    <Accordion.Root {...rest}>
+      <Accordion.Item value="one">
+        <Accordion.Header
+          data-h2-flex-grow="base(1)"
+          data-h2-display="base(flex)"
+          data-h2-flex-direction="base(row)"
+          data-h2-gap="base(x.5 0)"
+        >
+          <Accordion.Trigger>
+            <div
+              data-h2-flex-grow="base(1)"
+              data-h2-display="base(flex)"
+              data-h2-flex-direction="base(row)"
+              data-h2-gap="base(x.5 0)"
+            >
+              <div
+                data-h2-flex-grow="base(1)"
+                data-h2-display="base(flex)"
+                data-h2-flex-direction="base(column)"
+                data-h2-gap="base(x.5 0)"
+              >
+                <span
+                  data-h2-display="base(block)"
+                  data-h2-font-size="base(h6, 1)"
+                >
+                  <span data-h2-font-weight="base(700)">Custom</span>
+                  {" Title"}
+                </span>
```

Comment:
I understand this is just a story for examples sake, but an [accordion trigger should always contain an element with `role=heading`](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/#:~:text=Each%20accordion%20header%20button%20is%20wrapped%20in%20an%20element%20with%20role%20heading%20that%20has%20a%20value%20set%20for%20aria%2Dlevel%20that%20is%20appropriate%20for%20the%20information%20architecture%20of%20the%20page.).

Reply:
Yeah, this is helpful.  It is more than an example - I plan to use this for the new experience accordions.

Reply:
The radix header adds this role itself, doesn't it?
![image](https://user-images.githubusercontent.com/8978655/234597428-f20ebfc5-a805-4888-8d55-8367c1e10350.png)